src/main/java/fi/otavanopisto/kuntaapi/server/integrations/ptv/PtvOrganizationEntityUpdater.java

```diff
@@ -72,6 +83,8 @@ public TimerService getTimerService() {
   }
   
   private void executeNextTask() {
+    System.out.println("NEEEXT!");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)